Flip card creation

PJSalter opened the pull request:
Hey Sonia

Services page has it's own container that contains classes for the three flip cards produced

I've added the feature so that when the user hovers over the card then flips, instead of clicking on the card 🤩 This was super fun to make

I've got the cards to flip, edited with media queries so they should work for all screens.

I've added the colour scheme and box shadowing

Text is still just place holder that can be adjusted later

I hope you like it

let me know what you think, or if you think something needs to be changed 

⭐

sonianb commented on styles.css:
```diff
@@ -3,4 +3,110 @@ body{
+}
+
+.front-card-creativity{
+   font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```
For our website to be consistent, I'd suggest we decide on the main font and we use it for the entire page rather than having different fonts in different sections :) 
What do you think? When we decide on the font, we can add it to the root.

PJSalter replied:
Agree completely, once we choose a font it'll be consistent

sonianb commented on styles.css:
```diff
@@ -3,4 +3,110 @@ body{
+    }
+}
+
+@media only screen and (max-width: 425gpx) {
```
Is gpx a typo?

PJSalter replied:
Yes it is, thank you for spotting it 😸

sonianb commented on styles.css:
```diff
@@ -3,4 +3,110 @@ body{
+.detailed-info{
+    vertical-align: middle;
+    padding: 1rem;
+    font-size: 8px;
```
I noticed that you prefer using pixels over other units, which might not be the best choice for accessibility. 
Because if someone increases the font size on their screen, the size of the rest of the elements will remain the same when you use pixels. So maybe I'd be best practice to use rem or em units in places where you need to set a fixed width or height, since these units are relative to the font size.

PJSalter replied:
That's a very good point, ill change to only using 'rem' or 'em' 🌟

sonianb reviewed (changes requested):
I think there's a typo in your CSS file, check my comment.
Other than that, we're good to go! 😄

PJSalter commented:
Hi Sonia, I've fixed the typo and updated all the CSS elements with px to rem instead 🙂 I had some trouble with swapping between branches so I'll merge this to main now, so it's all updated
